tests/macros/t18235.nim

```diff
@@ -0,0 +1,10 @@
+discard """
+errormsg: '''
+undeclared identifier: 'foo'
+'''
+"""
+import m18235
```

Comment:
combine this test inside t18235_rename_export.nim and call:

doAssert not declared(foo)